cli/printers/table_printer.py

```diff
@@ -0,0 +1,123 @@
+                detections_per_detection_type_id[detection.detection_type_id].append(detection)
+
+        return detections_per_detection_type_id
+
+    def _print_detection_per_detection_type_id(self, detections_per_detection_type_id: Dict[str, Detection]):
+        for detection_type_id in list(detections_per_detection_type_id):
+
+            detections = detections_per_detection_type_id[detection_type_id]
+            if len(detections) == 0:
+                continue
+
+            headers = [REPOSITORY_COLUMN] if self._is_repository() else []
+
+            if detection_type_id == PACKAGE_VULNERABILITY_POLICY_ID:
+                headers = [SEVERITY_COLUMN] + headers
+                headers = headers + PREVIEW_DETECTIONS_COMMON_HEADERS[:]
+                headers.append(UPGRADE_COLUMN)
+                self._print_table_detections(detections,
+                                             headers,
+                                             self._get_upgrade_package_vulnerability,
+                                             "Dependencies Vulnerabilities")
+
+            if detection_type_id == LICENSE_COMPLIANCE_POLICY_ID:
+                headers = headers + PREVIEW_DETECTIONS_COMMON_HEADERS[:]
+                headers.append(LICENSE_COLUMN)
+                self._print_table_detections(detections,
+                                             headers,
+                                             self._get_license,
+                                             "License Compliance")
+
+    def _print_table_detections(self, detections: List[Detection], headers: List[str],
+                                get_row, title: str):
+        self._print_summary_issues(detections, title)
+        text_table = Texttable()
+        text_table.header(headers)
+        header_width_size_cols = []
+        for header in headers:
+            header_width_size_cols.append(len(header))
+        text_table.set_cols_width(header_width_size_cols)
+        for detection in detections:
+            row = get_row(detection)
+            text_table.add_row(row)
+        click.echo(text_table.draw())
+
+    def _print_summary_issues(self, detections: List, title: str):
+        click.echo(
+            f'⛔ Found {len(detections)} issues of type: {click.style(title, bold=True)}')
+
+    def _get_common_detection_fields(self, detection: Detection):
+        row = [
+            detection.detection_details.get('file_name'),
+            detection.detection_details.get('ecosystem'),
+            detection.detection_details.get('package_name'),
+            detection.detection_details.get('is_direct_dependency_str'),
+            detection.detection_details.get('is_dev_dependency_str')
+        ]
+
+        if self._is_repository():
+            row = [detection.detection_details.get('repository_name')] + row
+
+        return row
+
+    def _is_repository(self):
+        return self.context.obj.get("remote_url") is not None
```

Comment:
change the condition to - if its not path scan its repository scan.

Reply:
but even it's path scan the path could be repository